classes/Village.php

```diff
@@ -8,13 +8,19 @@ class Village {
 
     public string $name;
     public string $kage_name;
+    public int $region_id;
+    public int $village_id;
 
+    // to-do: we should restructure how village data is being saved
+    // player village should reference the village ID and this constructor should get row by ID
     public function __construct($system, $village) {
         $this->system = $system;
 
         $this->name = $village;
         $this->kage_name = $this->getKageName();
         $this->coords = $this->setVillageCoords();
+        $this->region_id = $this->getRegionId($this->system, $this->name);
+        $this->village_id = $this->getVillageId($this->system, $this->name);
```

Comment:
Not the biggest deal, but where possible let's do things in a single DB query. Could we query for both values here and set them in the constructor?

Reply:
Fixed.